src/test/java/nl/altindag/log/LogCaptorShould.java

```diff
@@ -359,6 +361,37 @@ void filterInfoMessages() {
                 .allMatch(Level.INFO::equals, "INFO");
     }
 
+    @Test
+    void captureMdcHeadersWhereLog4jIsUsed() {
+        logCaptor = LogCaptor.forClass(ServiceWithApacheLog4jAndMdcHeaders.class);
+
+        new ServiceWithApacheLog4jAndMdcHeaders().sayHello();
```

Comment:
This is a small detail, and again a personal preference... I like the inline way of calling it but would it be possible to have it similar to the other unit tests? So something like:
```
Service service = new ServiceWithApacheLog4jAndMdcHeaders();
service.sayHello();
```

Reply:
Done